metadata-ingestion/src/datahub/ingestion/source/state/stale_entity_removal_handler.py

```diff
@@ -34,6 +34,12 @@ class StatefulStaleMetadataRemovalConfig(StatefulIngestionConfig):
         default=True,
         description=f"Soft-deletes the entities of type {', '.join(_entity_types)} in the last successful run but missing in the current run with stateful_ingestion enabled.",
     )
+    fail_safe_threshold: float = pydantic.Field(
+        default=20.0,
```

Comment:
imo this should be like 95%

If everything was deleted, then we should fail. But if most stuff was removed, then it was probably just a refactor or pattern update.